README.md

```diff
@@ -369,22 +371,24 @@ Compose.prototype.map = function(f) {
 #### `traverse` method
 
 ```hs
-traverse :: Applicative f, Traversable t => t a ~> (a -> f b, c -> f c) -> f (t b)
+traverse :: Applicative f, Traversable t => t a ~> (TypeRep f, a -> f b) -> f (t b)
 ```
 
 A value which has a Traversable must provide a `traverse` method. The `traverse`
 method takes two arguments:
 
-    u.traverse(f, of)
+    u.traverse(A, f)
```

Comment:
Well for me, the TR is the more stable of the two parameters. It defines a classification if you will.

I could have a whole family of `a -> f b` functions, but they all tie to one `f`.

So using some point-free `traverse`, it just seems natural to fix that parameter and get a function back for that fixed Applicative. If that makes sense?